tests/js/spec/components/organizations/globalSelectionHeader.spec.jsx

```diff
@@ -426,9 +426,14 @@ describe('GlobalSelectionHeader', function() {
       initializationObj.routerContext
     );
 
-    expect(globalActions.updateProjects).not.toHaveBeenCalled();
-    expect(globalActions.updateEnvironments).not.toHaveBeenCalled();
-    expect(globalActions.updateDateTime).not.toHaveBeenCalled();
+    expect(globalActions.updateProjects).toHaveBeenCalledWith([]);
+    expect(globalActions.updateEnvironments).toHaveBeenCalledWith([]);
+    expect(globalActions.updateDateTime).toHaveBeenCalledWith({
+      end: null,
+      period: '14d',
+      start: null,
+      utc: null,
+    });
```

Comment:
If loading from the store is disabled, should this trigger?

Reply:
The idea was to make the URL parameters/store reflect user actions more closely so if end/start/period are null, it means the user has not selected a date.

Reply:
@billyvg @markstory  I've changed the behaviour so that if the datetime selection wasn't provided, the `period`/`statsPeriod` are now coerced to `14d` and injected into the store.

Hence why these store actions are now being called.

The `disableLoadFromStore` prop doesn't exist; would this particular test still be relevant?

Reply:
I would just remove it